tests/system/small/test_multiindex.py

```diff
@@ -367,6 +367,10 @@ def test_multi_index_dataframe_groupby_level_aggregate(
         .groupby(level=level, as_index=as_index)
         .mean(numeric_only=True)
     )
+    print("pandas")
```

Comment:
Stray debug statements (should be deleted)?

Reply:
removed